app/services/translations/de.ts

```diff
@@ -15,6 +15,19 @@ export const de = {
   errorMessages: {
     TRY_LATER_MESSAGE: "Bitte versuchen Sie es später erneut.",
     LOGIN_ERROR_MESSAGE: "Fehler bei der Anmeldung",
+    UNKNOWN_PAGE_LABEL: "Fehler 404",
+    UNKNOWN_PAGE_HEADING: "Seite konnte nicht gefunden werden",
+    UNKNOWN_PAGE_BODY:
+      "Die von Ihnen gewünschte Seite ist leider nicht verfügbar. Dies kann verschiedene Ursachen haben. \nWenn Sie die URL direkt eingegeben haben, überprüfen Sie die Schreibweise.\nVersuchen Sie, die Seite von der Startseite aus erneut zu finden.",
+    SERVER_ERROR_LABEL: "Fehler 500",
+    SERVER_ERROR_HEADING: "Ein interner Serverfehler ist aufgetreten",
+    SERVER_ERROR_BODY:
+      "Der Fehler liegt auf unserer Seite. Wir arbeiten bereits an einer Lösung. \nFalls das Problem bestehen bleibt, kontaktieren Sie unseren Support.",
+    GENERIC_ERROR_LABEL: "Ein Fehler ist aufgetreten",
+    GENERIC_ERROR_HEADING: "Etwas ist schiefgelaufen",
+    GENERIC_ERROR_BODY: "Bitte versuchen Sie es später erneut.",
+    VERFAHREN_ERROR_MESSAGE:
```

Comment:
NIT: I would suggest that we use something that may scale a bit more in the future for API related errors, for example...

```suggestion
    API_GET_VERFAHREN_ERROR_MESSAGE:
```
We can adjust this also later.